### PR TITLE
[proxima-direct-io-cassandra] O2-Czech-Republic#240 add seqId to cassandra

### PR DESCRIPTION
**Pull request by je-ik**

### CODE REVIEW

File: direct/io-cassandra/pom.xml [hunk 1]
```diff
@@ -119,10 +119,16 @@
...
+      <groupId>cz.o2.proxima</groupId>
+      <artifactId>proxima-direct-io-serialization</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
     <dependency>
       <groupId>com.datastax.cassandra</groupId>
       <artifactId>cassandra-driver-core</artifactId>
-      <version>3.8.0</version>
+      <version>3.11.0</version>
```
Comment (LesTR) [inline direct/io-cassandra/pom.xml:131]: NIT: we should probably bump this separately just to have clean history

**Review by LesTR — CHANGES_REQUESTED** [review]
I think we should move it out of direct module otherwise LGTM in general except some changes which look unrelated to this (probably needs to be rebased on top of master).

File: direct/io-cassandra/pom.xml (hunk 1 above)
Comment (je-ik) [reply direct/io-cassandra/pom.xml]: moved

File: direct/io-serialization/pom.xml [hunk 2]
```diff
@@ -0,0 +1,154 @@
+<!--
```
[the remark this answers is not in the document]
Comment (je-ik) [reply direct/io-serialization/pom.xml]: makes sense :+1:

File: direct/io-cassandra/src/main/java/cz/o2/proxima/direct/cassandra/Offsets.java [hunk 3]
```diff
@@ -20,6 +20,8 @@
 /** Offset based on {@code token} function. */
 public class Offsets {
 
+  private static RandomOffset EMPTY = new RandomOffset() {};
```
Comment (LesTR) [inline direct/io-cassandra/src/main/java/cz/o2/proxima/direct/cassandra/Offsets.java:23]: NIT: this can be also final

**Review by LesTR — APPROVED** [review]
👍

**Pull request merged**